products/centreon.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Centreon
+category: server-app
+tags: php-runtime
+permalink: /centreon
+versionCommand: cat /etc/centreon/centreon.conf
+releasePolicyLink: https://docs.centreon.com/docs/releases/lifecycle/
+releaseDateColumn: true
+activeSupportColumn: true
+eolColumn: Security Support
+
+#identifiers:
+#-   repology: zabbix
+#-   purl: pkg:brew/zabbix
+#-   purl: pkg:github/zabbix/zabbix
+
+auto:
+-   git: https://github.com/centreon/centreon.git
+
+# For non-LTS releases :
+# - support(x) = release(x) + 6 months
+# - eol(x) = release(x) + 7 months
+releases:
+-   releaseCycle: "23"
```

Comment:
Should be 23.04. According to https://docs.centreon.com/docs/releases/lifecycle/ there are two release per year named after the year and the month. Same for other versions.